tests/functional/cypress/e2e/form-settings.cy.js

```diff
@@ -0,0 +1,160 @@
+import 'cypress-keycloak-commands';
+import 'cypress-drag-drop';
+
+const depEnv = Cypress.env('depEnv');
+
+
+Cypress.Commands.add('waitForLoad', () => {
+  const loaderTimeout = 60000;
+
+  cy.get('.nprogress-busy', { timeout: loaderTimeout }).should('not.exist');
+});
+
+
+
+describe('Form Designer', () => {
+
+  beforeEach(()=>{
+    
+    cy.viewport(1000, 1800);
+    cy.waitForLoad();
+    cy.kcLogout();
+    cy.kcLogin("user");
+    
+    cy.on('uncaught:exception', (err, runnable) => {
+      // Form.io throws an uncaught exception for missing projectid
+      // Cypress catches it as undefined: undefined so we can't get the text
+      console.log(err);
+      return false;
+    });
+  });
+    
+// Verifying fields in the form settings page
+ it('Visits the form settings page', () => {
+   
+    cy.visit(`/${depEnv}/form/create`);
+    cy.location('pathname').should('eq', `/${depEnv}/form/create`);
+    cy.contains('h1', 'Form Settings');
+    cy.get('.v-row > :nth-child(1) > .v-card > .v-card-title > span').contains('Form Title');
```

Comment:
I'm just curious: should we be putting `data-cy="form-settings-title"` (or similar) attributes on component elements?

Reply:
@WalterMoar No, Its not putting by us. These attributes are from common components.